cmd/kops/create_cluster.go

```diff
@@ -86,6 +87,8 @@ func init() {
 	cmd.Flags().StringVar(&createCluster.AdminAccess, "admin-access", "", "Restrict access to admin endpoints (SSH, HTTPS) to this CIDR.  If not set, access will not be restricted by IP.")
 
 	cmd.Flags().BoolVar(&createCluster.AssociatePublicIP, "associate-public-ip", true, "Specify --associate-public-ip=[true|false] to enable/disable association of public IP for master ASG and nodes. Default is 'true'.")
+
+	cmd.Flags().StringVarP(&createCluster.Manifest, "manifest", "f", "", "Optional .yml cluster manifest")
```

Comment:
My thoughts..

All CRUD operations should use :

--filename=[]: Filename, or directory of a file identifying a .yml manifest.